pkg/operator/ceph/pool/controller.go

```diff
@@ -138,6 +148,11 @@ func poolChanged(old, new cephv1.PoolSpec) bool {
 }
 
 func (c *PoolController) onDelete(obj interface{}) {
+	if c.clusterSpec.External.Enable && !c.clusterSpec.External.RunStatelessWorkload {
+		logger.Warningf("Deleting pools for an external ceph cluster is disabled (RunStatelessWorkload is false)")
```

Comment:
Nit: from the user-facing side of things, this is `runStatelessWorkload` not capitalized.